.travis.yml

```
@@ -53,7 +53,7 @@ jobs:
   include:
     - <<: *doc_test
       os: linux
-      julia: 1.0
+      julia: 1
```

I'd like to ensure that the documentation tests still succeed against the earliest version of Julia supported by this package.
```suggestion
      julia: 1.0
```